src/tiled/commanddialog.cpp

```diff
@@ -63,6 +69,24 @@ void CommandDialog::closeEvent(QCloseEvent *event)
     CommandManager::instance()->updateActions();
 }
 
+void CommandDialog::setShortcut(const QKeySequence &keySequence)
+{
+    if (!mUi->keySequenceEdit->isEnabled())
+        mUi->keySequenceEdit->setEnabled(true);
```

Comment:
Why is this here, though? First of all there is no point in checking whether the sequence edit is disabled before setting it to enabled, but the main thing I'm wondering is why should it get enabled in this function? I think this should happen in `updateKeySequenceEdit`.